src/utils/types/index.ts

```diff
@@ -51,6 +53,10 @@ export const gameSchema = z.object({
   answerKey: z.string().url().optional(),
   videoTrailer: z.string().url().optional(),
 });
+//Since arrays from req.query are just strings, and need to be converted into arrays.
+
+//For validating game query in the GET endpoint
+//Be aware that zod.transform alters the data during parsing.
```

Comment:
I'm kinda confused what these comments mean, can they be removed?

Reply:
Yeah they can be removed, a lot of these comments are me just thinking through the code before writing it.